components/renku_data_services/group/api.spec.yaml

```diff
@@ -270,146 +273,134 @@ components:
         id: "01AN4Z79ZS5XN0F25N3DB94T4R"
         name: "R-Project Group"
         slug: "r-project"
-        created_by:
-          id: "owner-KC-id"
-    GroupPost:
+        created_by: "owner-keycloak-id"
+    GroupPostRequest:
       description: Group metadata to be created in Renku
       type: object
       additionalProperties: false
       properties:
         name:
-          $ref: "#/components/schemas/Name"
+          $ref: "#/components/schemas/GroupName"
         slug:
-          $ref: "#/components/schemas/Slug"
+          $ref: "#/components/schemas/GroupSlug"
         description:
-          $ref: "#/components/schemas/Description"
+          $ref: "#/components/schemas/GroupDescription"
       required:
         - "name"
-    GroupPatch:
+        - "slug"
+    GroupPatchRequest:
       type: object
       description: Patch of a group
       additionalProperties: false
       properties:
         name:
-          $ref: "#/components/schemas/Name"
+          $ref: "#/components/schemas/GroupName"
         slug:
-          $ref: "#/components/schemas/Slug"
+          $ref: "#/components/schemas/GroupSlug"
         description:
-          $ref: "#/components/schemas/Description"
+          $ref: "#/components/schemas/GroupDescription"
     Ulid:
       description: ULID identifier
       type: string
       minLength: 26
       maxLength: 26
       pattern: "^[A-Z0-9]{26}$"  # This is case-insensitive
       format: ulid
-    Name:
+    GroupName:
       description: Renku group name
       type: string
       minLength: 1
       maxLength: 99
       example: "My Renku Group :)"
-    Slug:
+    GroupSlug:
```

Comment:
Do we want to have a type name for each kind of slug we have (e.g., `ProjectSlug`, `GroupSlug`...)? If we can keep the structure of all slugs the same (and I think we should do this), then I would favor having just one `Slug` type, ideally with it defined only once in some shared location.